zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -1,4 +1,27 @@
-//! Instructions provided by the ZkToken Proof program
+//! Instructions provided by the [`ZK Token proof`] program.
+//!
+//! There are two types of instructions in the proof program: the proof verification instructions
+//! and the `CloseContextState` instruction.
+//!
+//! Each proof verification instruction verifies a certain type of zero-knowledge proof. These
+//! instructions are processed by the program in two steps:
+//!   1. The program verifies the zero-knowledge proof.
+//!   2. The program optionally stores the context data component of the instruction data to a
```

Comment:
```suggestion
//!   2. The program optionally stores the context component of the instruction data to a
```
Does this still make sense? It seemed like too many "data"s